### PR TITLE
Working features in browser

### PR DESCRIPTION
**Pull request by colinbruce**
#### What
@lostie implemented chrome based testing as part of #2225

#### Why
Not all tests would run successfully, some because of the use of `.trigger('click')` some because of hidden or hover elements that poltergeist ignores.

#### How
Find all uses of `.trigger('click')` and replace with `.click`, this is honoured by poltergeist and chrome-webdriver

Identified and rectified all other failures.  

#### TODO (wip)
 - [ ] There is a possibility that one of the tests will flicker.  Monitor and address as needed

### CODE REVIEW

File: features/step_definitions/advocate_claim_steps.rb [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
 Then(/^I select the first search result$/) do
   sleep Capybara.default_max_wait_time
-  @claim_form_page.offence_results.first.select_button(visible: false).trigger('click')
+  find(:xpath, '//*[@id="offence-list"]/div[3]/div').hover
```
Comment (jsugarman) [inline features/step_definitions/advocate_claim_steps.rb:43]: As discussed, can we just put a `# TODO: get working in site-prism` on this.

**Review by jsugarman — APPROVED** [review]
👍 just that todo comment

File: features/step_definitions/advocate_claim_steps.rb (hunk 1 above)
Comment (colinbruce) [reply features/step_definitions/advocate_claim_steps.rb:43]: I've added an issue, #2454

**Comment by lostie** [comment]
🏆 😻

**Pull request merged**